contrib/modelci/README.md

```diff
@@ -0,0 +1,143 @@
+# Model CI
+
+The users occasionally found a negligible performance or precision issue between different Paddle versions. Though we have unit tests for each class and Travis-CI to ensures the precision of each operator, there is no any logic to ensure the model (a composition of several operators) works as reliable as the operators.
+
+There are several conditions where an existing model will fail either in performance or precision:
+
+1. the incomplete coverage test cases. 
+2. poor performance update, currently, we have no performance tracker for each operator.
+3. API changes, developers are likely to forget to update usages in some other repositories such as paddle/models.
```

Comment:
there is another benefit:
We can point user to our benchmark models as "currently working examples"

Reply:
Agree, some classical models will be tested every time.

Some more popular models might be tested every several days, and the ModelCI will generate a report about the precision and performance of the PaddlePaddle platform.